### PR TITLE
feat: break copy link

### PR DESCRIPTION
**Pull request by ciyer**
Support breaking the link to the template project

## Screenshot

image

## Testing

- Create a project
- Use the copy feature to copy the project
- On the copy, go to the settings page and fill out the form to unlink the project

After that, the project should no longer show a link to the template

/deploy renku=release-0.62.0

### CODE REVIEW

File: client/src/features/ProjectPageV2/ProjectPageContent/Settings/ProjectUnlinkTemplate.tsx [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+    <>
+      Project <code>{projectName}</code> successfully unlinked.
+    </>
+  );
+}
+
+interface ProjectUnlinkTemplateProps {
+  project: Project;
+}
+export default function ProjectUnlinkTemplate({
```
Comment (leafty) [inline client/src/features/ProjectPageV2/ProjectPageContent/Settings/ProjectUnlinkTemplate.tsx:47]: Why does this section exist when the project is not linked anymore? It also appears on a brand new project created from scratch.

Comment (ciyer) [reply client/src/features/ProjectPageV2/ProjectPageContent/Settings/ProjectUnlinkTemplate.tsx:47]: Oops -- had a `=== null` check where a ` == null` was intended. Fixed now.

File: client/src/features/ProjectPageV2/ProjectPageContent/Settings/ProjectUnlinkTemplate.tsx [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+        template_id: "",
+      },
+    });
+  }, [patchProject, project.etag, project.id]);
+
+  useEffect(() => {
+    if (result.isSuccess) {
+      if (notifications)
+        notificationProjectDeleted(notifications, project.name);
+    }
```
Comment (leafty) [inline client/src/features/ProjectPageV2/ProjectPageContent/Settings/ProjectUnlinkTemplate.tsx:67]: I think you should reset the `result` here, the notification keeps re-appearing when I open/close the new project modal from the top bar.

**Review by leafty — COMMENTED** [review]
This looks good 👍!

There is on comment below about a small issue about the notifications.

File: client/src/features/ProjectPageV2/ProjectPageContent/Settings/ProjectUnlinkTemplate.tsx (hunk 2 above)
Comment (ciyer) [reply client/src/features/ProjectPageV2/ProjectPageContent/Settings/ProjectUnlinkTemplate.tsx:66]: You are right, this has been fixed.

**Review by leafty — APPROVED** [review]
LGTM

**Pull request merged**